doc/changes/changes_1.0.0.md

```diff
@@ -0,0 +1,11 @@
+# error-reporting-lua 1.0.0, released yyyy-mm-dd
+
+Code name: MVP
```

Comment:
don't abreviate

Reply:
Looks like I taught you well. Was curious if you would find this. :-)